.github/workflows/system-tests.yml

```diff
@@ -84,11 +84,10 @@ jobs:
           path: artifact.tar.gz
 
   parametric:
-    needs:
-      - build-artifacts
-    uses:  DataDog/system-tests/.github/workflows/system-tests.yml@main
+    uses: DataDog/system-tests/.github/workflows/run-parametric.yml@main
     secrets: inherit
     with:
-      scenarios: PARAMETRIC
       library: nodejs
       binaries_artifact: system_tests_binaries
+      _experimental_job_count: 8
+      _experimental_job_matrix: '[1,2,3,4,5,6,7,8]'
```

Comment:
are those pytest arguments ? or where are they from ? where are htey documented ?

Reply:
They are converted to pytest arguments. Not documented anywhere, it's part of system-tests repo.